src/Servers/ShellServer.cs

```diff
@@ -184,21 +184,6 @@ public virtual void OnKernelInfoRequest(Message message)
                         }
                     }
                 );
-
-                // Finish by telling the client that we're free.
```

Comment:
It's not clear from the PR why this can be removed.

Reply:
Ah, I forgot to push one file, sorry. This was moved to KernelApplication.cs. Thanks for catching!